stores/auth.ts

```diff
@@ -98,6 +98,8 @@ export const useAuthStore = defineStore("auth", {
     },
   },
   persist: {
-    storage: persistedState.localStorage,
+    storage: persistedState.cookiesWithOptions({
```

Comment:
if i'm reading this right, we're now using cookies: have I missed the cookie banner somewhere? (or is it not necessary?)

Reply:
not necessary as its not used to track people in any way. additionally, a cookie banner would have been necessary for localstorage already...